scriptmodules/supplementary/runcommand/runcommand.sh

```diff
@@ -1101,6 +1101,14 @@ function retroarch_append_config() {
         iniSet "video_fullscreen_y" "${dim[1]}"
     fi
 
+    # set `libretro_directory` to the core parent folder
+    local core_dir=$(echo "$COMMAND" | grep -Eo " $ROOTDIR/libretrocores/.*libretro\.so " | head -n 1)
+    core_dir=$(dirname "$core_dir")
+    [[ -n "$core_dir" ]] && iniSet "libretro_directory" "$core_dir"
+
+    # if verbose logging is on, set core logging to INFO
+    [[ $COMMAND =~ "--verbose" ]] && iniSet "libretro_log_level" "1"
```

Comment:
Might be safer/easier to set a global variable when we add the --verbose to avoid the comparison.

Reply:
Makes sense - I added a global var, similar to how `$NETPLAY` is set.